TestRail_Navigation_C+.md

```diff
@@ -0,0 +1,30 @@
+# TestRail Navigation and Process for C+
+
+## Context
+C+ are now responsible for finding the appropriate location for a new or updated regression test case before the proposed test steps are posted by the Contributor who fixed the bug. This document is intended to give a short overview on how to navigate TestRail and best practices on choosing the correct location for the proposed steps. 
+As a C+ member, you will have view-only access to Expensify's TestRail account in order to complete this task.     
```

Comment:
Maybe we can add quick steps - if they don't have access to TestRail yet, how can they ask for it? (probably just asking in #contributor-plus)?

Reply:
> C+ are now responsible for

C+ are responsible for

Reply:
Love it! @Beamanator I'm gonna work on getting C+ into TestRail before we launch, and then include adding them to TestRail as part of their onboarding, so I think we're good on that? But if you disagree say so!

@Julesssss good catch!